src/App.tsx

```
@@ -55,16 +57,24 @@ function AppContent() {
 
   // NEØ OAuth2 Bridge - Inicialização crítica (versão React Hook)
-  const { authorize, checkSession, isReady } = useOAuth2Bridge();
-  const { isSynced, hasValidAuth } = useAuthSync();
+  // ✅ SOLUÇÃO: Usar mock em desenvolvimento
+  const isDevelopment = window.location.hostname === 'localhost' || window.location.hostname === '127.0.0.1';
+  
+  const { authorize, checkSession, isReady } = isDevelopment 
+    ? { authorize: async () => ({ success: true }), checkSession: async () => ({ success: true }), isReady: true }
+    : useOAuth2Bridge();
+    
+  const { isSynced, hasValidAuth } = isDevelopment
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)